frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -57,11 +57,11 @@ pods:
               ./bootstrap && ./{{HDFS_VERSION}}/bin/hdfs journalnode
         resource-set: journal-resources
         configs:
-          {{#KERBEROS_ENABLED}}
+          {{#SECURITY_KERBEROS_ENABLED}}
           krb5-conf:
             template: krb5.conf
             dest: {{JRE_VERSION}}/lib/security/krb5.conf
```

Comment:
Is it *required* that this be copied to `{{JRE_VERSION}}/lib/security/krb5.conf`? This breaks as soon as we update the JRE version. Would it not be more scalable to use `{{HDFS_VERSION}}/etc/security/krb5.conf` here?

Reply:
There's already a JIRA for this, so I might as well just fix it here.

Reply:
Sure, let's not do it here. The other ticket will be fine.